includes/classes/class-rest-api.php

```diff
@@ -124,23 +125,6 @@ function () use ( $ns, $version ) {
 			}
 		);
 
-		add_action(
-			'rest_api_init',
-			function () use ( $ns, $version ) {
-				register_rest_route(
-					$ns . $version,
-					'/scans-stats-by-post-type/(?P<slug>[a-zA-Z0-9_-]+)',
-					array(
-						'methods'             => 'GET',
-						'callback'            => array( $this, 'get_scans_stats' ),
-						'permission_callback' => function () {
-							return current_user_can( 'read' ); // able to access the admin dashboard.
-						},
-					)
-				);
-			}
-		);
-
```

Comment:
Bugfix - not needed. Closes: https://github.com/equalizedigital/accessibility-checker/pull/365